config/main.py

```diff
@@ -2064,6 +2064,538 @@ def delete_snmptrap_server(ctx, ver):
     cmd="systemctl restart snmp"
     os.system (cmd)
 
+
+
+#
+# 'snmp' group ('config snmp ...')
+#
+@config.group(cls=clicommon.AbbreviationGroup, name='snmp')
+@clicommon.pass_db
+def snmp(db):
+    """SNMP configuration tasks"""
+
+
+@snmp.group(cls=clicommon.AbbreviationGroup)
+@clicommon.pass_db
+def community(db):
+    pass
+
+
+def is_valid_community_type(commstr_type):
+    commstr_types = ['RO', 'RW']
+    if commstr_type not in commstr_types:
+        click.echo("Invalid community type.  Must be either RO or RW")
```

Comment:
>click.echo [](http://example.com/codeflow?start=8&length=10)

I see you have `click.echo` both inside and outside this checker function. The inner one seems redundant.

Reply:
You can just
```
return commstr_type in commstr_types
```

Reply:
Using helper function for the comments per my discussion with Qi.